code/game/atoms.dm

```diff
@@ -1769,3 +1769,25 @@
 /atom/proc/InitializeAIController()
 	if(ai_controller)
 		ai_controller = new ai_controller(src)
+
+/**
+ * Point at an atom
+ *
+ * Intended to enable and standardise the pointing animation for all atoms
+ *
+ * Not intended as a replacement for the mob verb
+ */
+/atom/proc/pointat(atom/A)
+	if(!src || !isturf(src.loc))
```

Comment:
```suggestion
	if(!isturf(loc))
```
If there's no source the proc won't run.
No need to add the `src.` if there's no local (proc) variable with the same name.